server/static/gateway.css

```diff
@@ -0,0 +1,191 @@
+
+/* HEADER NAVIGATION */
+.navbar {
+	min-height: 48px;
+	padding: 0 12px;
+	border-radius: 0;
+}
+
+.navbar-default {
+	border-color: rgba(0,0,0,0);
+	background-color: #fff;
+	box-shadow: 0 3px 4px 0 rgba(0,0,0,.2), 0 3px 3px -2px rgba(0,0,0,.14), 0 1px 8px 0 rgba(0,0,0,.12);
+}
+.navbar-default .navbar-brand,
+.navbar-default .navbar-brand:hover,
+.navbar-default .navbar-text,
+.navbar-default .navbar-nav>li>a {
+	color: rgba(0,0,0,0.87);
+}
+
+.navbar-default .navbar-brand {
+	padding: 14px 15px;
+}
+.navbar-default .navbar-nav>li>a {
+	text-transform: uppercase;
+	font-size: 13px;
+}
+.navbar-default .navbar-nav>li>a:hover {
+	background-color: transparent;
+}
+.navbar-default .navbar-form {
+	margin: 10px 0;
+}
+.login-link,
+.login-link:hover {
+	font-size: 12px;
+	padding: 5px 10px;
+	text-decoration: none;
+	color: rgba(0,0,0,0.87);
+	border-radius: 4px;
+	border: 1px solid rgba(0,0,0,.26);
+}
+
+/* MAIN CONTENT AREA */
+body {
+	font-family: "Roboto","Helvetica","Arial",sans-serif;
+}
+
+.container {
+	padding: 0;
+}
+
+h3 {
+	color: rgba(0,0,0,.87);
+    font-family: "Roboto","Helvetica","Arial",sans-serif;
```

Comment:
this is same as body definition. are you overriding font-family somewhere else that this needs to be here because of that? If not, you probably should be able to just remove this.

Reply:
haha im actually not sure, the UX gave me this css file =) 
I would just leave it..